omod/src/main/java/org/openmrs/module/rwandahivflowsheet/web/model/HivFlowsheetFormData.java

```diff
@@ -196,18 +196,18 @@ public Date getArvStartDate() {
     	
     	for (DrugOrder dor : rawOrders){
     		//drug start dates:
-    		Set<DrugOrder> dorListOnStartDate = ret.get(dor.getStartDate());
+    		Set<DrugOrder> dorListOnStartDate = ret.get(dor.getEffectiveStartDate());
     		if (dorListOnStartDate == null)
     			dorListOnStartDate = new LinkedHashSet<DrugOrder>();
     		//now check all orders to see if they're active on that date also, and add if so:
     		for (DrugOrder dorInner : rawOrders){
-    			if (dorInner.isCurrent(dor.getStartDate()) && !dorListOnStartDate.contains(dorInner)) //only add each DrugOrder once.
+    			if (dorInner.isActive(dor.getEffectiveStartDate()) && !dorListOnStartDate.contains(dorInner)) //only add each DrugOrder once.
```

Comment:
You may need to do a sanity check here to ensure that "isActive" is comparable to "isCurrent" previously.

Reply:
Saw the page at https://docs.openmrs.org/doc-1.10/org/openmrs/Order.html#isActive-java.util.Date- and thought #isActive might be the right one